# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/runit/rockify.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

we need to export vars to be read inside init.d/system
currently we simulate sysv's stage 2
  Alejandro

--- ./etc/debian/1.orig	2003-11-18 12:47:03.000000000 -0300
+++ ./etc/debian/1	2003-12-07 10:28:44.000000000 -0300
@@ -1,10 +1,16 @@
 #!/bin/sh
 # system one time tasks
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+{
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-/etc/init.d/rcS
-/etc/init.d/rmnologin
+/etc/rc.d/init.d/system start
+#/etc/rc.d/init.d/network start
+
+# write ET mark for btee
+echo -ne '\004'
+
+} 2>&1 | /sbin/btee a /var/log/init.msg
 
 touch /etc/runit/stopit
 chmod 0 /etc/runit/stopit
--- ./etc/debian/3.orig	2003-11-18 12:47:03.000000000 -0300
+++ ./etc/debian/3	2003-12-07 10:28:44.000000000 -0300
@@ -1,10 +1,10 @@
 #!/bin/sh
 exec 2>&1
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-LAST=0
-test -x /etc/runit/reboot && LAST=6
+export RUNLEVEL=0
+test -x /etc/runit/reboot && export RUNLEVEL=6
 
 echo 'Waiting for getties to stop...'
 svwaitdown -xk -t14 /var/service/getty-*
@@ -13,4 +13,4 @@
 svwaitdown -xk -t350 /var/service/*
 
 echo 'Shutdown...'
-/etc/init.d/rc $LAST
+/etc/rc.d/init.d/system stop
